package.json

```diff
@@ -15,6 +15,7 @@
   ],
   "dependencies": {},
   "devDependencies": {
+    "eslint-plugin-prettier": "^3.1.4",
```

Comment:
is this dependency needed?

Reply:
Without this line, `npm test` will fail.

Reply:
That is surprising, `xo` is the `eslint` wrapper and it is supposed to automatically bundle all its dependencies.
What error are you seeing when this dependency isn't explicitly added?

Reply:
@uetchy could you try 1) removing this line, 2) removing (nested) `node_modules`, 3) `npm install`ing again? And see if it works now?

Reply:
I tried:

1. `rm -rf **/node_modules`
1. `yarn`
1. `node_modules/.bin/xo --fix`

and got this:

```
Error: Failed to load plugin 'prettier' declared in 'CLIOptions': Cannot find module 'eslint-plugin-prettier'
Require stack:
- /Users/uetchy/Repos/src/github.com/remarkjs/remark-math/__placeholder__.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1085:15)
    at Function.resolve (internal/modules/cjs/helpers.js:81:19)
    at Object.resolve (/Users/uetchy/Repos/src/github.com/remarkjs/remark-math/node_modules/eslint/lib/shared/relative-module-resolver.js:28:50)
    at ConfigArrayFactory._loadPlugin (/Users/uetchy/Repos/src/github.com/remarkjs/remark-math/node_modules/eslint/lib/cli-engine/config-array-factory.js:992:39)
    at /Users/uetchy/Repos/src/github.com/remarkjs/remark-math/node_modules/eslint/lib/cli-engine/config-array-factory.js:883:33
    at Array.reduce (<anonymous>)
    at ConfigArrayFactory._loadPlugins (/Users/uetchy/Repos/src/github.com/remarkjs/remark-math/node_modules/eslint/lib/cli-engine/config-array-factory.js:879:22)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/Users/uetchy/Repos/src/github.com/remarkjs/remark-math/node_modules/eslint/lib/cli-engine/config-array-factory.js:707:44)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (/Users/uetchy/Repos/src/github.com/remarkjs/remark-math/node_modules/eslint/lib/cli-engine/config-array-factory.js:647:20)
```

and realized that I only get this error when I use `yarn`. I'll use `npm` for this PR as it has no issue around `xo`.

Reply:
fixed in https://github.com/remarkjs/remark-math/pull/48/commits/237fbe8599d6d420c54d7ab6d0d7a84e77b22c45